packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -266,7 +268,7 @@ describeNoCompat("blobs", (getTestObjectProvider) => {
         if (provider.driver.type !== "odsp") {
             // this flow is currently only supported on ODSP, the others should explicitly reject on attach
             return assert.rejects(attachP,
-                (err) => /(0x202)|(0x204)/.test(err.message) /* "create empty file not supported" */);
+                (err) => err.message === usageErrorMessage /* "create empty file not supported" */);
```

Comment:
nit (here and elsewhere):
```suggestion
                (err) => err.message === usageErrorMessage);
```